dcrwallet.go

```diff
@@ -109,6 +109,8 @@ func walletMain() error {
 	loader := ldr.NewLoader(activeNet.Params, dbDir, stakeOptions, cfg.UnsafeMainNet,
 		cfg.AddrIdxScanLen, cfg.AllowHighFees, cfg.RelayFee.ToCoin())
 
+	passphrase := &[]byte{}
+	defer zero.Bytes(*passphrase)
```

Comment:
this still looks wrong, it is still referring to the same bytes, and when zeroed, the ticket buyer won't be able to unlock the wallet.

and please change it back to a slice, not a pointer to a slice